pom.xml

```diff
@@ -94,8 +94,10 @@
 		<package.group>${project.artifactId}</package.group>
 		<package.install.dir>/usr/share/${project.artifactId}</package.install.dir>
 		<package.user>${project.artifactId}</package.user>
+                <package.userId>102</package.userId>
```

Comment:
I'm going to show my ignorance, but how do we ensure that `102` does not conflict with an existing package? A quick look into `/usr/share/doc/setup-*/uidgid` indicates that `102` is currently free, but this does not seem to be future proof...